javaslang/src/main/java/javaslang/collection/Traversable.java

```diff
@@ -447,6 +447,21 @@ default T single() {
     <C> Map<C, ? extends Traversable<T>> groupBy(Function<? super T, ? extends C> classifier);
 
     /**
+     * Matches each element with a unique key that you extract from it.
+     * If the same key is present twice, the function will return {@code None}.
+     *
+     * @param classifier A function which extracts a key from elements
+     * @param <K>        key class type
+     * @return A Map containing the elements arranged by their keys.
+     * @throws NullPointerException if {@code getKey} is null.
```

Comment:
Could you please add a `@see #groupBy(Function)` to the javadoc of arrangeBy()
and a `@see #arrangeBy(Function)` to the javadoc of groupBy()?

I think it is good to promote the new method because it is not known from other libraries.